Fixed hireling Bolt ID, Demon outfit addon 1, updated drowning and others

AdversarioV opened the pull request:
Previous bolt id for hireling was 3483 which was a fishing rod, changed it for the proper one 3446


# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Behaviour
### **Actual**

Do this and that doesn't happens

### **Expected**

Do this and that happens

## Fixes

\# (issue)

## Type of change

Please delete options that are not relevant.

  - [ ] Bug fix (non-breaking change which fixes an issue)
  - [ ] New feature (non-breaking change which adds functionality)
  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - [ ] This change requires a documentation update

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

  - [ ] Test A
  - [ ] Test B

**Test Configuration**:

  - Server Version:
  - Client:
  - Operating System:

## Checklist

  - [ ] My code follows the style guidelines of this project
  - [ ] I have performed a self-review of my own code
  - [ ] I checked the PR checks reports
  - [ ] I have commented my code, particularly in hard-to-understand areas
  - [ ] I have made corresponding changes to the documentation
  - [ ] My changes generate no new warnings
  - [ ] I have added tests that prove my fix is effective or that my feature works

lBaah commented on data/XML/mounts.xml:
```diff
@@ -20,7 +20,7 @@
 	<mount id="19" clientid="402" name="Tamed Panda" speed="20" premium="yes" />
 	<mount id="20" clientid="405" name="Dromedary" speed="20" premium="yes" />
-	<mount id="21" clientid="406" name="Scorpion King" speed="20" premium="yes" />
+	<mount id="21" clientid="406" name="Sandstone Scorpion" speed="20" premium="yes" />
```
https://tibia.fandom.com/wiki/Scorpion_King

It's Scorpion King actually.

CoehManeh replied:
The mount that tames is the Sandstone Scorpion
https://www.tibiawiki.com.br/wiki/Sandstone_Scorpion

EnzzoCaaue replied:
sanstone scorpion is the name of the creature but the mount name is scorpion king

lBaah commented:
Possible conflict with #700

MajestyOTBR commented:
**This branch has conflicts that must be resolved**
Use the **web editor** or the **command line** to resolve conflicts. 

**Conflicting files**
data/actions/scripts/others/mounts/mounts.lua

EnzzoCaaue commented on data/events/scripts/player.lua:
```diff
@@ -4,7 +4,7 @@ GOLD_POUCH = 26377
 
 DISABLE_CONTAINER_WEIGHT = 0 -- 0 = YES, ENABLE | 1 = NO, DISABLE
-CONTAINER_WEIGHT = 100000 -- 10k = 10000 oz | this function is only for containers, item below the weight determined here can be moved inside the container, for others items look game.cpp at the src
+CONTAINER_WEIGHT = 10000000 -- 100.000,00 oz | this function is only for containers, item below the weight determined here can be moved inside the container, for others items look game.cpp at the src
```
dunno why change it to 100k

lBaah replied:
A better thing would be the count of items inside containers because i can threw trash with 0.01oz and load thousand of stackable items, such gold coin in 1 per slot.

take a look at https://github.com/otland/forgottenserver/pull/2659

MajestyOTBR commented on data/items/items.xml:
```diff
@@ -44717,4 +44723,10 @@ Her... li.. .. S... ura... erim (The engraving is old and almost unreadable with
+	<item id="34784" article="a" name="mysterious scroll">
+		<attribute key="description" value="Strange symbols hard to decrypt. You wonder if one of the symbols looks like a foreign and mysterious creature." />
+		<attribute key="weight" value="850" />
+	</item>
```
Can you put this item in order by id please?

AdversarioV committed:
Fixed Hireling bolt ID

Previous bolt id for hireling was 3483 which was a fishing rod, changed it for the proper one 3446

AdversarioV committed:
Fixed the ID for bolts

AdversarioV committed:
So this doesn't quite fix the issue however it does remove the drowning from someone if they wear a special helmet

AdversarioV committed:
Merge branch 'main' of https://github.com/AdversarioV/otservbr-global

AdversarioV committed:
Updated datapack

Update from mainbranch

Commit:
Updated to version v1.3.65

AdversarioV committed:
Fix demon outfit shield

fix demon outfit shield addon

Commit:
Updated to version v1.3.66

AdversarioV committed:
Merge branch 'main' into main
